src/frontend/apps/lti_site/components/LocalizedTimedTextTrackUpload/index.spec.tsx

```diff
@@ -50,15 +49,24 @@ describe('<LocalizedTimedTextTrackUpload />', () => {
     });
   });
 
-  it('renders the component without any uploaded timed text track', () => {
+  afterEach(() => {
```

Comment:
I get lot of errors when I launch the test, considere to mock console.error or mock the missing translation.

```
console.error
    Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "components.LanguageSelect.selectLanguageLabel" for locale "fr-FR", using default message as fallback.
```